examples/amp-first/pages/index.js

```diff
@@ -231,6 +189,48 @@ const Home = props => (
         </section>
       </main>
     </Layout>
+    <style jsx>{`
```

Comment:
@timneutkens should css always go last? Or only when using fragments?

Reply:
I'd always put it last otherwise it generates some cognitive overhead in understanding the structure of the html